src/server/rpc/procedures/chart/gnuPlotTest.js

```diff
@@ -0,0 +1,55 @@
+const gnuPlot = require('./node-gnuplot.js'),
```

Comment:
If this is a testing file, can you move it to `test/`?

Reply:
it is not an automated test so where would be good place to put it?